go.mod

```diff
@@ -81,6 +81,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/vishvananda/netlink v1.1.0
 	github.com/vmware/govmomi v0.20.3
+	go.etcd.io/etcd/api/v3 v3.5.0
```

Comment:
note, i cannot approve changes in the go.mod.
once we are done with the review here we can assign one of the maintainers for go.mod changes.